build-llvm.py

```diff
@@ -639,10 +639,12 @@ def generate_pgo_profiles(args, dirs):
     utils.print_header("Building PGO profiles")
 
     # Run kernel/build.sh
-    subprocess.run(
-        [dirs.root_folder.joinpath("kernel", "build.sh"), '-t', args.targets],
-        check=True,
-        cwd=dirs.build_folder.as_posix())
+    subprocess.run([
+        dirs.root_folder.joinpath("kernel", "build.sh"), '-b',
+        dirs.build_folder, '-t', args.targets
+    ],
```

Comment:
If you are separating the lines, maybe it looks better to put an option per line, i.e. something like:

    subprocess.run([
        dirs.root_folder.joinpath("kernel", "build.sh"),
        '-t', dirs.build_folder,
        '-b', args.targets
    ],

Reply:
I agree that looks better but I've been formatting the code with [yapf](https://github.com/google/yapf) for consistency and it chose this look. It reformats your suggestion to the current look.

Reply:
Never mind then! :)